packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -324,6 +325,8 @@ class SnackBar extends StatefulWidget {
   /// If this property is null, then [SnackBarThemeData.insetPadding] of
   /// [ThemeData.snackBarTheme] is used. If that is also null, then the default is
   /// `EdgeInsets.fromLTRB(15.0, 5.0, 15.0, 10.0)`.
+  ///
+  /// If this property is not null, then [HitTestBehavior.deferToChild] is used by default.
```

Comment:
It seems there is something missing in this comment because it implies that `margin` will be set to `[HitTestBehavior.deferToChild]`.

```dart
/// If this property is not null and [hitTestBehavior] is null, then [hitTestBehavior] default is [HitTestBehavior.deferToChild].
```